FordLabs/PeopleMover
# 147 remove old ux

## Issue
Resolves 147 (Make newui the default UI, removing the old UI experience)

## What was done
- [X] Remove conditional code and feature tag
- [X] Remove old UI components (and tests) no longer being used

## How to test
1. Open PeopleMover. Note that the #newui feature tag experience is the default experience. #newui doesn't change the layouts anymore
